maint/check-code-format.sh

```diff
@@ -47,9 +47,10 @@ pushd "${THIS_SCRIPT_DIR}/.."
 
 declare ARGON_SRC_DIR="build/third-party/argon_transformer/src/ext_argon_transformer/src"
 declare ARGON_TEST_DIR="build/third-party/argon_transformer/src/ext_argon_transformer/test"
+declare PYBIND_WRAPPER="python/pyngraph"
 
 declare ROOT_SUBDIR
-for ROOT_SUBDIR in src test ${ARGON_SRC_DIR} ${ARGON_TEST_DIR}; do
+for ROOT_SUBDIR in src test ${ARGON_SRC_DIR} ${ARGON_TEST_DIR} ${PYBIND_WRAPPER}; do
```

Comment:
Aren't these wrappers generated code? We shouldn't be formatting generated code.